Makefile

```diff
@@ -282,6 +283,24 @@ dist_xpi: add-on_dist
 	rm -f $(DIST)/$(XPI_NAME)
 	@$(REPO_BIN_DIR)/build_extension.sh $(XPI_NAME) $(DIST) add-on
 
+.PHONY: dist_export
+dist_export:
+	rm -rf $(DIST_EXPORT_DIR)
+	# Do a basic copy of the add-on.
+	$(RSYNC) $(BUILT)/$(ADD-ON)/* $(DIST_EXPORT_DIR)
+	# Use the install.rdf.in rather than install.rdf.
+	rm -f $(DIST_EXPORT_DIR)/install.rdf
+	$(RSYNC) $(ADD-ON)/install.rdf.in $(DIST_EXPORT_DIR)
+	# jar.mn is used for Firefox build
+	rm -f $(DIST_EXPORT_DIR)/chrome.manifest
+	# Add the jar file.
+	$(RSYNC) $(ADD-ON)/jar.mn $(DIST_EXPORT_DIR)
+	# Use the original prefs.js.
+	$(RSYNC) $(ADD-ON)/preferences/prefs.js $(DIST_EXPORT_DIR)/chrome/content/preferences/prefs.js
+	# Copy the vendor files that support the unit tests.
+	$(RSYNC) $(BUILT)/test/vendor/* $(DIST_EXPORT_DIR)/chrome/content/shared/test/vendor
```

Comment:
This is for export to mozilla-central. If we don't export unit tests, then we can't have them running against the firefox builds.

The add-on dist goes to a separate location.